lib/src/executable/options.dart

```diff
@@ -78,6 +79,34 @@ class ExecutableOptions {
       ..addFlag('embed-source-map',
           help: 'Embed source map contents in CSS.', defaultsTo: false);
 
+    parser
+      ..addSeparator(_separator('Warnings'))
+      ..addFlag('quiet', abbr: 'q', help: "Don't print warnings.")
+      ..addFlag('quiet-deps',
+          help: "Don't print compiler warnings from dependencies.\n"
+              "Stylesheets imported through load paths count as dependencies.")
+      ..addFlag('verbose',
+          help: "Print all deprecation warnings even when they're repetitive.")
+      ..addMultiOption('fatal-deprecation',
+          help: 'Deprecations to treat as errors. You may also pass a Sass\n'
+              'version to include any behavior deprecated in or before it.\n'
+              'See https://sass-lang.com/documentation/breaking-changes for \n'
```

Comment:
We should update this documentation so that each breaking change includes an example of how to make it fatal as well

Reply:
Working on a sass-site PR now

Reply:
https://github.com/sass/sass-site/pull/702